src/game/enemies/marun.c

```diff
@@ -20,7 +21,7 @@ typedef struct {
     /* 0x0A */ s8 direction;
     /* 0x0C */ u16 region[2];
     /* 0x10 */ u16 rotation;
-    /* 0x12 */ u16 unk12;
+    /* 0x12 */ u16 timer;
```

Comment:
Needs to be s16, apparently.
Unless that makes it not match in functions other than sub_8063ADC, of course.